src/components/Profile/Profile.js

```diff
@@ -63,11 +71,99 @@ class Profile extends Component {
                         </Row>
                     </Col>
                     <Col>
-                        {this.loadReservations()}
+                        {
+                            this.mock_reservations.map((eachReservation) => {
+                            if( eachReservation.memberID === this.memberID ) {
+                                return (
+                                    <div key={ eachReservation.reservation_key }>
+                                        { eachReservation.reservation.map((eachComponent, index) => {
+                                        return (
+                                            <div key={ index }>
+                                                <Row className={ (index % 2 === 0 ? "oddRow" : "evenRow") }>
+                                                    <Col className="reservations">{eachReservation.date}</Col>
+                                                    <Col className="reservations">{eachComponent.componentID}</Col>
+                                                    <Col className="reservations">{eachComponent.quantity}</Col>
+                                                </Row>
+                                            </div>
+                                        )
+                                        })}   
+                                    </div>
+                                )
+                            }
+                            return null;
+                            })
+                        }
                     </Col>
+                </div>
+            )
+        } else {
+            return (
+                <h3>You have not done any reservations</h3>
+            )
+        }
+    }
+
+    /*
+    Returns the table that contains returned components
+    */
+   loadReturned() {
```

Comment:
Code should be self-documented, trying to explain it with names rather than comments. Try using more descriptive names for variables and functions
```suggestion
   loadReturnedComponentsTable() {
```